### PR TITLE
docs: Document CredentialFile and CredentialJson options

### PR DESCRIPTION
**Pull request by jskeet**

### CODE REVIEW

File: apis/Google.Cloud.Logging.Log4Net/docs/configuration.md [hunk 1]
```diff
@@ -123,7 +127,24 @@ If `resourceType` has been manually set, then `resourceLabel` is used to specify
 
 Defaults to `false`. Setting this to `true` disables automatic `resourceType` setting based on platform detection.
 
-### Complete configuration example
+### CredentialFile
+
+```xml
+<credentialFile value="path/to/credential.json" />
+
+Specifies the path to a JSON file containing credentials.
```
Comment (Rishabh-V) [inline apis/Google.Cloud.Logging.Log4Net/docs/configuration.md:135]: Do we need to close the xml code snippet? ```

File: apis/Google.Cloud.Logging.Log4Net/docs/configuration.md [hunk 2]
```diff
@@ -123,7 +127,24 @@ If `resourceType` has been manually set, then `resourceLabel` is used to specify
...
+
+### CredentialJson
+
+```xml
+<credentialJson value="{ inline JSON credentials }" />
+
```
Comment (Rishabh-V) [inline apis/Google.Cloud.Logging.Log4Net/docs/configuration.md:141]: Here as well.

**Review by Rishabh-V — APPROVED** [review]
Just a couple of questions.

File: apis/Google.Cloud.Logging.Log4Net/docs/configuration.md (hunk 1 above)
Comment (jskeet) [reply apis/Google.Cloud.Logging.Log4Net/docs/configuration.md:135]: Doh, yes thanks!

**Comment by jskeet** [comment]
Fixed XML snippets - will merge on green.

**Pull request merged**